### PR TITLE
Make client constructors no-fail

### PR DESCRIPTION
**Pull request by jhendrixMSFT**
Moved parsing out of the constructor, into the API call.
Fixed an issue with imports not getting added to client.go files.

### CODE REVIEW

File: test/network/2020-03-01/armnetwork/vpnsites.go [hunk 1]
```diff
@@ -161,11 +164,15 @@ func (client *vpnSitesOperations) ResumeDelete(token string) (HTTPPoller, error)
...
+	u, err := url.Parse(client.u)
+	if err != nil {
+		return nil, err
+	}
 	urlPath := "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/vpnSites/{vpnSiteName}"
 	urlPath = strings.ReplaceAll(urlPath, "{subscriptionId}", url.PathEscape(client.subscriptionID))
 	urlPath = strings.ReplaceAll(urlPath, "{resourceGroupName}", url.PathEscape(resourceGroupName))
 	urlPath = strings.ReplaceAll(urlPath, "{vpnSiteName}", url.PathEscape(vpnSiteName))
-	u, err := client.u.Parse(path.Join(client.u.Path, urlPath))
+	u, err = u.Parse(urlPath)
```
Comment (catalinaperalta) [inline test/network/2020-03-01/armnetwork/vpnsites.go:175]: Just double checking here, this will append urlPath to the already existing path on the url?

Comment (jhendrixMSFT) [reply test/network/2020-03-01/armnetwork/vpnsites.go:175]: Good catch, no it doesn't work as expected.  Will fix.

Comment (jhendrixMSFT) [reply test/network/2020-03-01/armnetwork/vpnsites.go]: Fixed

**Pull request merged**